rviz_ogre_vendor/package.xml

```diff
@@ -28,7 +28,8 @@
   <build_export_depend>libglew-dev</build_export_depend>
   <exec_depend>libfreetype6</exec_depend>
 
-  <depend>libx11-dev</depend>
+  <depend condition="$XDG_SESSION_TYPE == 'x11'">libx11-dev</depend>
+  <depend condition="$XDG_SESSION_TYPE == 'wayland'">libwayland-dev</depend>
```

Comment:
How will this affect the binaries (.debs) provided by ROS? Or any others built by a CI system that has no `$XDG_SESSION_TYPE` set?

It seems like CI generated binaries, esp. those provided by the ROS project, should be built with compatibility in mind (i.e support both platforms if possible)?

It certainly seems like it would be undesirable for a headless CI system to result in it building with _neither_ X11 nor Wayland support 😁